core/src/main/java/io/snappydata/impl/SnappyHiveCatalog.java

```diff
@@ -270,8 +270,7 @@ public Object call() throws Exception {
               Table table = hmc.getTable(schema, tableName);
               String tableType = table.getParameters().get(
                   JdbcExtendedUtils.TABLETYPE_PROPERTY());
-              if (!ExternalTableType.Row().name().equalsIgnoreCase(tableType) &&
-                  !ExternalTableType.Index().name().equalsIgnoreCase(tableType)) {
+              if (!ExternalTableType.Row().name().equalsIgnoreCase(tableType)) {
```

Comment:
It will now show up in SYS.HIVETABLES but have not added it to getTables. It should be added to SYS.INDEXES or similar APIs at some point (the GUI can use SYS.HIVETABLES to show them in any case).